gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.2-bin.zip
```

Comment:
this should be all

Reply:
I'll believe this comment: https://github.com/DerBanko/BoardGames/pull/2#issuecomment-774537783